RecursiveFileList.py

```diff
@@ -0,0 +1,86 @@
+#!/bin/env python3
+
+"""Print a map of the directory/file structure of a given base path.
+The module will print out a full map in dictionary format, a list of the files with their full paths,
+and a list of directories with their full paths. The module can be run using the following command:
+
+python3 RecursiveFileList.py <absolute path>
+
+It can also be imported by another python program and used like:
+
+#import RecursiveFileList
+RecursiveFileList.map_dir("<absolute path>")
```

Comment:
`map_dir` takes two arguments, an enclosing folder and a folder within it -- so instead I would suggest two examples like:
``` RecursiveFileList.map_dir("<absolute path to enclosing folder> / <subfolder>")```
and
``` RecursiveFileList.map_dir("", "<absolute path to folder>")```